client/src/api.ts

```diff
@@ -351,14 +362,41 @@ export function getNotifications(
   return secureFetch(`${API_BASE}/notifications?${params}`);
 }
 
-export function getQrCodes(
+export async function getQrCodes(
   limit?: number,
   offset?: number,
   status?: boolean,
   event_id?: number
 ): Promise<PaginatedQrCodes> {
   const params = queryString.stringify({ limit, offset, status, event_id }, { sort: false });
-  return secureFetch(`${API_BASE}/qr?${params}`);
+  return secureFetch(`${API_BASE}/qr-code?${params}`);
+}
+
+export async function qrCodesRangeAssign(
+  from: number,
+  to: number,
+  eventId: number | null
```

Comment:
This `eventId` can be `null`? Or it's optional / maybe undefined?

Reply:
event can be null because of the backend implementation. To unassign range of qrs from an event, you should send the range with the event null

Reply:
Great, thanks for the explanation!